static/app/components/events/interfaces/noStackTraceMessage.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from '@emotion/styled';
+
+import Alert from 'app/components/alert';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+
+type Props = {
+  message?: React.ReactNode;
+};
+
+function NoStackTraceMessage({message}: Props) {
+  return (
+    <StyledAlert type="error">
+      <i>{message ?? t('No or unknown stacktrace')}</i>
+    </StyledAlert>
+  );
+}
+
+export default NoStackTraceMessage;
+
+const StyledAlert = styled(Alert)`
+  border-color: ${p => p.theme.border};
+  padding: ${space(1)} ${space(3)};
+  font-size: ${p => p.theme.fontSizeMedium};
```

Comment:
Why change the border color? This looks weird compared to all our other alerts

Reply:
I thought it would be better if the component did not look too different from this: https://github.com/getsentry/sentry/blob/4c8da8997e37c2643a16aa0ed5a12f53fe8622ec/static/app/components/events/interfaces/threads/content.tsx#L49

Reply:
Should be we aligning that error and this one to our more frequently used alert styles instead of making additional copies of styles that aren't frequently used?